### PR TITLE
Feature/cli push

### PR DESCRIPTION
**Pull request by panasevychol**
## Proposed changes

Push CLI func implemented.

## Types of changes

What types of changes does your code introduce to agents-aea?
_Put an `x` in the boxes that apply_

- [ ] Bugfix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)

## Checklist

_Put an `x` in the boxes that apply._

- [ ] I have read the [CONTRIBUTING](../master/CONTRIBUTING.rst) doc
- [ ] I am making a pull request against the `develop` branch (left side). Also you should start your branch off our `develop`.
- [ ] Lint and unit tests pass locally with my changes
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] I have checked that code coverage does not decrease.
- [ ] I have checked that the documentation about the `aea cli` tool works
- [ ] I have added necessary documentation (if appropriate)
- [ ] Any dependent changes have been merged and published in downstream modules

### CODE REVIEW

File: aea/cli/push.py [hunk 1]
```diff
@@ -0,0 +1,50 @@
...
+"""Implementation of the 'aea push' subcommand."""
+import click
+
+from aea.cli.registry.push import push_item
+
+
+@click.group()
+def push():
+    """Push item to Registry."""
+    pass
```
[the remark this answers is not in the document]
Comment (panasevychol) [reply aea/cli/push.py]: @DavidMinarsch just defining a group of commands, no action needed. Should I leave a comment about that?

File: aea/cli/registry/push.py [hunk 2]
```diff
@@ -0,0 +1,88 @@
...
+        f.add(source_dir, arcname=os.path.basename(source_dir))
+
+
+def push_item(item_type: str, item_name: str) -> None:
+    item_type_plural = item_type + 's'
+    cwd = os.getcwd()
+
+    items_folder = os.path.join(cwd, item_type_plural)
+    item_path = os.path.join(items_folder, item_name)
+    click.echo(
```
[the remark this answers is not in the document]
Comment (panasevychol) [reply aea/cli/registry/push.py]: @DavidMinarsch do you think so? It's just an echo message to let user know on what stage of pushing he's at this moment. It's not a debug log.

File: aea/cli/push.py (hunk 1 above)
[the remark this answers is not in the document]
Comment (DavidMinarsch) [reply aea/cli/push.py]: Yes I would add a comment

File: aea/cli/registry/push.py (hunk 2 above)
[the remark this answers is not in the document]
Comment (DavidMinarsch) [reply aea/cli/registry/push.py]: Ok, the issue is, no user wants to know all that. Its useful for debugging, but for normal usage there should be just one message: either successfully pushed or had an error..

[the remark this answers is not in the document]
Comment (panasevychol) [reply aea/cli/registry/push.py]: @DavidMinarsch got it. I can clean this and other messages.

File: aea/cli/registry/push.py [hunk 3]
```diff
@@ -37,16 +37,21 @@ def _compress(output_filename: str, source_dir: str):
...
+    :param item_name: str item name.
+
+    :return: None
+    """
     item_type_plural = item_type + 's'
     cwd = os.getcwd()
 
     items_folder = os.path.join(cwd, item_type_plural)
     item_path = os.path.join(items_folder, item_name)
-    click.echo(
```
Comment (DavidMinarsch) [inline aea/cli/registry/push.py]: I would not remove them, rather than change to `logger.debug()`

File: aea/cli/registry/push.py [hunk 4]
```diff
@@ -56,14 +61,10 @@ def push_item(item_type: str, item_name: str) -> None:
         )
 
     output_filename = '{}.tar.gz'.format(item_name)
-    click.echo('Compressing {} {} to {} ...'.format(
```
Comment (DavidMinarsch) [inline aea/cli/registry/push.py]: `logger.debug()` same

File: aea/cli/registry/push.py [hunk 5]
```diff
@@ -56,14 +61,10 @@ def push_item(item_type: str, item_name: str) -> None:
...
-        item_name, item_type, output_filename
-    ))
     _compress(output_filename, item_path)
     output_filepath = os.path.join(cwd, output_filename)
 
     item_config_filepath = os.path.join(item_path, '{}.yaml'.format(item_type))
-    click.echo('Reading {} {} config ...'.format(item_name, item_type))
```
Comment (DavidMinarsch) [inline aea/cli/registry/push.py]: `logger.debug()` same

File: aea/cli/registry/push.py [hunk 6]
```diff
@@ -72,9 +73,6 @@ def push_item(item_type: str, item_name: str) -> None:
         'version': item_config['version']
     }
     path = '/{}/create'.format(item_type_plural)
-    click.echo('Pushing {} {} to Registry ...'.format(
```
Comment (DavidMinarsch) [inline aea/cli/registry/push.py]: `logger.info()`this can be shown to the user

File: aea/cli/registry/push.py [hunk 7]
```diff
@@ -83,6 +81,3 @@ def push_item(item_type: str, item_name: str) -> None:
             item_type, item_name, resp['public_id']
         )
     )
-
-    click.echo('Removing temporary file {}'.format(output_filepath))
```
Comment (DavidMinarsch) [inline aea/cli/registry/push.py]: same `logger.debug()`

**Pull request merged**